pkg/network/usm/ebpf_main.go

```diff
@@ -112,6 +111,26 @@ func newEBPFProgram(c *config.Config, sockFD, connectionProtocolMap *ebpf.Map, b
 		},
 	}
 
+	if c.CollectTCPv4Conns || c.CollectTCPv6Conns {
+		missing, err := ddebpf.VerifyKernelFuncs("sockfd_lookup_light")
+		if err == nil && len(missing) == 0 {
+			mgr.Probes = append(mgr.Probes, []*manager.Probe{
+				{
+					ProbeIdentificationPair: manager.ProbeIdentificationPair{
+						EBPFFuncName: probes.SockFDLookup,
+						UID:          probeUID,
+					},
+				},
+				{
+					ProbeIdentificationPair: manager.ProbeIdentificationPair{
+						EBPFFuncName: probes.SockFDLookupRet,
+						UID:          probeUID,
+					},
+				},
+			}...)
+		}
```

Comment:
if the kernel does not have such function, we won't have any log, and we will not have any idea why core functionality in USM does not work

Reply:
This was the case before as well. I can add a log, but I am not sure what else we can do here.

Reply:
What would you like the log message to say? I am not sure how adversely USM would be affected.

Reply:
@guyarb

Reply:
re: affect - it can result in missing TLS requests
if it didn't have log until today - I'm fine with keeping it as is in the PR (as the PR just transfers ownership) we can revisit the solution in a dedicated PR and better address it in USM